schema.prisma

```diff
@@ -209,10 +209,22 @@ model Campaign {
   incomingTransfers       Transfer[]     @relation("target_campaign")
   vaults                  Vault[]
   withdrawals             Withdrawal[]
+  slugArchive             SlugArchive[]
 
   @@map("campaigns")
 }
 
+/// Keeps track of previous slugs that are not used currently in any active campaign
+model SlugArchive {
+ id                      String         @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+ slug                    String         @unique @db.VarChar(250)
```

Comment:
if the slug is UNIQUE then let's make it PRIMARY and we can go without the "id" column

Reply:
I have applied those changes